src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
         ["mg"] = new("Malagasy", "Malagasy", "mg", "mlg"),
-        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
-        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
-        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mhr"] = new("Eastern Mari", "олык марий", "mhr", "mhr", TranslationServices.Yandex),
         ["mi"] = new("Maori", "Te Reo Māori", "mi", "mri"),
-        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mk"] = new("Macedonian", "Македонски", "mk", "mkd"),
+        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
         ["mn"] = new("Mongolian", "Монгол хэл", "mn", "mon"),
+        ["mn-Mong"] = new("Mongolian (Traditional)", "ᠮᠣᠩᠭᠣᠯ ᠬᠡᠯᠡ", "mn-Mong", "mn-Mong", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mrj"] = new("Western Mari", "Мары йӹлмӹ", "mrj", "mrj", TranslationServices.Yandex),
+        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
+        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mww"] = new("Hmong Daw", "Hmong Daw", "mww", "mww", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ne"] = new("Nepali", "नेपाली", "ne", "nep"),
+        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
         ["no"] = new("Norwegian", "Norsk", "no", "nor"),
+        ["nso"] = new("Sepedi", "Sesotho sa Leboa", "nso", "nso", TranslationServices.Google),
+        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
+        ["om"] = new("Oromo", "Afaan Oromoo [ʌfɑːn ɔrɔmoː] Oromiff", "om", "orm", TranslationServices.Google),
         ["or"] = new("Odia", "ଓଡ଼ିଆ", "or", "ori", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["ps"] = new("Pashto", "پښتو", "ps", "pus", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["fa"] = new("Persian", "فارسی", "fa", "fas"),
+        ["otq"] = new("Querétaro Otomi", "Hñähñu", "otq", "otq", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["pa"] = new("Punjabi", "ਪੰਜਾਬੀ", "pa", "pan"),
+        ["pap"] = new("Papiamento", "Papiamento", "pap", "pap", TranslationServices.Yandex),
         ["pl"] = new("Polish", "Polski", "pl", "pol"),
+        ["prs"] = new("Dari", "دری", "prs", "prs", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ps"] = new("Pashto", "پښتو", "ps", "pus", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["pt"] = new("Portuguese", "Português", "pt", "por"),
-        ["pa"] = new("Punjabi", "ਪੰਜਾਬੀ", "pa", "pan"),
+        ["pt-BR"] = new("Portuguese (Brazil)", "Português (Brazil)", "pt-BR", "pt-BR", TranslationServices.Google),
+        ["qu"] = new("Quechua", "Kechua", "qu", "que", TranslationServices.Google),
```

Comment:
The native name should be "Runa simi".
```suggestion
        ["qu"] = new("Quechua", "Runa simi", "qu", "que", TranslationServices.Google),
```

Reply:
https://omniglot.com/language/names.htm
The above link shows Kechua as primary, and Runa simi as secondary. And to make it more complicated, when I translate the word "Quechua" to English, I get "Runasimi".

I've changed it to "Runa simi", but after the update, I'm going to change it to what the translation service gives "Runasimi".
I'm going to have it as Runasimi, because I'm adding a validation function that will validate that all the languages have the correct native name.

IMHO, the _languages shouldn't have the **native name** included. Instead this should be done programmatically. Why do something manually when the program can do this for you?

Reply:
> IMHO, the _languages shouldn't have the native name included. Instead this should be done programmatically. Why do something manually when the program can do this for you?

What do you mean? That the language list should be source-generated? That's something I'm planning to do.
Right now I can scrape the language list from the source code of Google Translate's page. This includes the English name and native name too. With Bing/Microsoft Azure this is easier as there's an API endpoint that provides that info. Unfortunately it's not possible to get the ISO 639-3 code from these methods but I guess there's a API somewhere that would provide that info.